GeoLib/Station.cpp

```diff
@@ -23,12 +24,12 @@
 namespace GeoLib
 {
 Station::Station(double x, double y, double z, std::string name) :
-    Point (x,y,z), _name(name), _type(Station::StationType::STATION),
+    Point (x,y,z), _name(std::move(name)), _type(Station::StationType::STATION),
```

Comment:
Because `name` is passed by value it is cheaper to move and we know that `std::string` has a move-ctor.
Maybe passing by const ref in first place would have been a better idea...

Changing parameter to const ref, because this is something we all understand well.